src/models/message.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @module models
+ */
+
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+const Joi = require('joi')
+Joi.objectId = require('joi-objectid')(Joi)
+
+// We create the Schema for chat, and we set up the required variables
+
+/**
+ * Answers schema, containing user, date, content
+ * @constructor Message
+ */
+const messageSchema = new Schema({
+  user: {
+    type: mongoose.Types.ObjectId,
+    ref: 'users',
+    required: true
+  },
+  date: {
+    type: Date,
+    default: new Date(),
+    required: true
+  },
+  content: {
+    type: String,
+    required: true
+  }
+})
+
+// We create answers collection from answersSchema
+const Messages = mongoose.model('messages', messageSchema)
```

Comment:
use camelCase

Reply:
This line follows the same rules as the others models
![image](https://github.com/SchoodEIP/Schood-API/assets/72021963/a8582ee1-2fbc-4231-b764-1a67b4675a0f)